README.md

```diff
@@ -0,0 +1,60 @@
+# kubeadm-systemd
+
+## Introduction
+
+__kubeadm-systemd__ is a tool for creating a multi-node Kubernetes cluster
+on the single machine, created mostly for developers __of__ Kubernetes.
+
+It aims to be as similar to the solutions recommendend for the production
+clusters as possible.
+
+## Architecture
+
+![Architecture Diagram](architecture.png?raw=true "Architecture")
+
+kubeadm-systemd uses the following third-party components to
+achieve its goal:
+
+### CNI
+
+Raw systemd-nspawn needs systemd-networkd for automanaging the networking
+for containers, which is bad for developers trying to use nspawn on their
```

Comment:
@alban probably. Maybe there is a way to use some dhcp server to get `--network-*` options, but it sounds worse to me than just using CNI.